apis/cyndi-operator/v1alpha1/cyndipipeline_types.go

```diff
@@ -0,0 +1,115 @@
+/*
```

Comment:
Is this and the next few files all copied over from the cyndipipeline operator? and if so, do we expect them to change and how should we keep them in sync? Is this because cyndi itself is private?

Reply:
Yes it is copied over from here: https://github.com/RedHatInsights/cyndi-operator/blob/master/api/v1alpha1/

With one change, I removed [this line](https://github.com/RedHatInsights/cyndi-operator/blob/master/api/v1alpha1/cyndipipeline_types.go#L32) because when I tried to 'make install' my k8s setup did not seem to approve of an optional bool with a default (which should have basically no consequence, since the empty value of a bool is `false` anyway)

Reply:
In future I'd like to *not* copy this, and hopefully we could just `go get` the cyndi operator instead. But yes, since the repo is private, I just copied this in.

Reply:
Sure - we should make sure that @we have some way of knowing when it's changed so we can sync it